code/go/0chain.net/chaincore/block/entity.go

```diff
@@ -944,6 +951,29 @@ func (b *Block) ComputeState(ctx context.Context, c Chainer) error {
 	return nil
 }
 
+func (b *Block) addTransactions(c Chainer) {
```

Comment:
You need to remove this & add something like emitAddTransaction, see this code block by @ssardana08 for miner https://github.com/0chain/0chain/blob/ffcf5a425aa7d917a68fee18b7363bd7ae0d7d26/code/go/0chain.net/smartcontract/minersc/sharder_eventdb.go#L76 , something similar needs to be done for transactions.

Reply:
done... discussed and a different approach was decided and implemented